blueprints/validator/files/tests/unit/validators/__name__-test.js

```diff
@@ -1,9 +1,8 @@
 import Ember from 'ember';
-import { moduleFor, test }
-from 'ember-qunit';
+import { moduleFor, test } from 'ember-qunit';
 
 moduleFor('validator:<%= dasherizedModuleName %>', 'Unit | Validator | <%= dasherizedModuleName %>', {
-  needs: ['validator:messages']
+  integration: true
```

Comment:
I definitely don't have a strong opinion on it, was just curious what necessitated the change. Integration tests are very helpful, though slightly slower.